src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -42,6 +42,16 @@ configurationRegistry.registerConfiguration({
 		'datasource.connectionGroups': {
 			'description': localize('datasource.connectionGroups', "data source groups"),
 			'type': 'array'
+		},
+		'datasource.connectionSort': {
+			'type': 'string',
+			'enum': ['byTimeAdded', 'byTitleAlphabetically'],
```

Comment:
'DateAdded', 'Name'

Reply:
And specifically I think he's suggesting `byDateAdded` and `byName` for the enum names. 

And maybe even `byDisplayName` to clearly indicate that it's the name shown in the OE tree - which is not necessarily the server name. 

(title is unclear what that means since we don't refer to title in the UI at all)

Reply:
updated
@Charles-Gagnon With the setting key updated to `sortBy` from `connectionSort`, I used `DateAdded` and `DisplayName` as enum names.